### PR TITLE
mediatek: add Comfast CF-E395AX support by adding an alternative model name

### PR DESCRIPTION
**Pull request by db260179**
both these devices share the board and same config, just different model number

Install instructions are the same as the CF-E393AX commit - https://git.openwrt.org/?p=openwrt/openwrt.git;a=commit;h=d8f4453bf2de9fd9baf3d660ed12e0797ff2cfdb

### CODE REVIEW

**Comment by Neustradamus** [comment]
@db260179: Good job!

File: target/linux/mediatek/dts/mt7981a-comfast-cf-e39xax-common.dtsi [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+		#size-cells = <0>;
+
+		port@0 {
+			reg = <0>;
+			label = "lan1";
+		};
+
+		port@1 {
+			reg = <1>;
+			label = "eth1";
```
Comment (hauke) [inline target/linux/mediatek/dts/mt7981a-comfast-cf-e39xax-common.dtsi:218]: Why did you add this `port@1` configuration?

File: target/linux/mediatek/dts/mt7981a-comfast-cf-e393ax.dts [hunk 2]
```diff
@@ -1,247 +1,9 @@
...
-	status = "okay";
-};
-
-&wifi {
-	status = "okay";
-	nvmem-cells = <&eeprom_factory_0>;
-	nvmem-cell-names = "eeprom";
-};
-
+};
```
Comment (hauke) [inline target/linux/mediatek/dts/mt7981a-comfast-cf-e393ax.dts:9]: Please add a new line at the end of the file and also the other files.

**Comment by hauke** [comment]
What is the difference between the two devices?

File: target/linux/mediatek/dts/mt7981a-comfast-cf-e39xax-common.dtsi (hunk 1 above)
Comment (db260179) [reply target/linux/mediatek/dts/mt7981a-comfast-cf-e39xax-common.dtsi:218]: So my other PR  - https://github.com/openwrt/openwrt/pull/16138 had identified that we where missing the config for the wan port.

This commit just aligns with that change

File: target/linux/mediatek/dts/mt7981a-comfast-cf-e393ax.dts (hunk 2 above)
Comment (db260179) [reply target/linux/mediatek/dts/mt7981a-comfast-cf-e393ax.dts:9]: Yep, will do

**Comment by db260179** [comment]
> What is the difference between the two devices?

Just the model number, reason for this common config - the chassis is literally the only difference - they cost reduced it, so the 393ax has a bigger case than the 395ax - to board is identical.

File: target/linux/mediatek/dts/mt7981a-comfast-cf-e39xax-common.dtsi (hunk 1 above)
Comment (hauke) [reply target/linux/mediatek/dts/mt7981a-comfast-cf-e39xax-common.dtsi:218]: Please do not hide such changes. When you do more than just moving the code add a extra commit

Comment (db260179) [reply target/linux/mediatek/dts/mt7981a-comfast-cf-e39xax-common.dtsi:218]: No intention of hiding anything?
Please advise what i should do? change this line and then do a second commit again?

Comment (db260179) [reply target/linux/mediatek/dts/mt7981a-comfast-cf-e39xax-common.dtsi]: @hauke As per your request, i have removed the seperate changes and will submit a seperate patch for those changes once this change is approved.

**Comment by hauke** [comment]
Do we really need an extra image and device tree?
If it is just about the name you can add an alternative name:
```
--- a/target/linux/mediatek/image/filogic.mk
+++ b/target/linux/mediatek/image/filogic.mk
@@ -482,6 +482,8 @@ TARGET_DEVICES += cmcc_rax3000m
 define Device/comfast_cf-e393ax
   DEVICE_VENDOR := COMFAST
   DEVICE_MODEL := CF-E393AX
+  DEVICE_ALT0_VENDOR := COMFAST
+  DEVICE_ALT0_MODEL := CF-E395AX
   DEVICE_DTS := mt7981a-comfast-cf-e393ax
   DEVICE_DTS_DIR := ../dts
   DEVICE_DTC_FLAGS := --pad 4096
```

**Comment by db260179** [comment]
> Do we really need an extra image and device tree? If it is just about the name you can add an alternative name:
> 
> ```
> --- a/target/linux/mediatek/image/filogic.mk
> +++ b/target/linux/mediatek/image/filogic.mk
> @@ -482,6 +482,8 @@ TARGET_DEVICES += cmcc_rax3000m
>  define Device/comfast_cf-e393ax
>    DEVICE_VENDOR := COMFAST
>    DEVICE_MODEL := CF-E393AX
> +  DEVICE_ALT0_VENDOR := COMFAST
> +  DEVICE_ALT0_MODEL := CF-E395AX
>    DEVICE_DTS := mt7981a-comfast-cf-e393ax
>    DEVICE_DTS_DIR := ../dts
>    DEVICE_DTC_FLAGS := --pad 4096
> ```

Yes makes sense, only issue the users might not know that this image will work on the e395ax, but i have changed the commit to your recommended change.

**Comment by blogic** [comment]
merged, Thanks !

**Comment by Neustradamus** [comment]
@blogic: Are you sure?

I do not see here:
- https://github.com/openwrt/openwrt/blob/main/target/linux/mediatek/image/filogic.mk

**Comment by Neustradamus** [comment]
Merged commit:
- https://github.com/openwrt/openwrt/commit/794291bbdf26ad2be7581fc0c921e4d820937c79
